sql/parser/builtins.go

```diff
@@ -331,12 +334,23 @@ var builtins = map[string][]builtin{
 			types:      typeList{},
 			returnType: DummyFloat,
 			impure:     true,
-			fn: func(args DTuple) (Datum, error) {
+			fn: func(_ EvalContext, args DTuple) (Datum, error) {
 				return DFloat(rand.Float64()), nil
 			},
 		},
 	},
 
+	"unique_id": {
```

Comment:
Using `uuid` in the name might be confusing if the returned value is not 16-bytes long and does not contain valid UUID version number and variant fields.